packages/config/src/layer2s/common/stages/types.ts

```diff
@@ -20,21 +20,30 @@ export type ChecklistTemplate<T extends StageBlueprint> = {
   }
 }
 
+// TODO: maybe it shouldn't be undefined
+export type Stage = 'Stage 0' | 'Stage 1' | 'Stage 2' | undefined
```

Comment:
'Candidate' ?

Reply:
Yeah, I was thinking about that, but we do not have any designs ready for this case. We could ignore it for now IMO